src/control-plane/backend/lambda/sfn-workflow/index.ts

```diff
@@ -147,6 +147,9 @@ async function stackParametersResolve(stack: WorkFlowStack) {
         ({ key, value } = await _getParameterKeyAndValueByJSONPath(param.ParameterKey, param.ParameterValue, bucket, prefix));
       } else if (param.ParameterKey?.endsWith('.#') && param.ParameterValue?.startsWith('#.')) { // Find the value in output by suffix
         ({ key, value } = await _getParameterKeyAndValueByStackOutput(param.ParameterKey, param.ParameterValue, bucket, prefix));
+      } else {
+        key = param.ParameterKey;
```

Comment:
how about moving it when initializing the variables `key` and `value`?

Reply:
done.